### ISSUE TITLE
Any way to keep login for days?

### ISSUE THREAD

**Issue by zhuaoyuan** [Chatie/wechaty#988]
### Provide Your Network Information

Where is the location of your server?  Shanghai
Which cloud platform(AliYun/Qcloud/DigitalOcean/etc) are you using? Tencent cloud

wechaty version:0.13.12
According to my observation, my bot deployed in docker logs out automatically every 30 hours(approximately), which is annoying :<. I guess the reason is that the session gets expired. Are there any available solutions?

**Comment by suntong** [comment]
@zhuaoyuan, is your session still only last 30 hours now, or it is only once or twice? Have you tried other network or cloud platform than Tencent cloud?

**Comment by zixia** [comment]
As my experience, we can keep login session for weeks without any problem.

I'll close this for now, but please feel free to file new issue if you can provide the reproduce steps for this issue.
